internal/pkgecosystem/crates.io.go

```diff
@@ -0,0 +1,40 @@
+package pkgecosystem
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type cratesIoJSON struct {
```

Comment:
So in Go's style guide it says that acronyms need to be capitalised.

`.io` stands for "British Indian Ocean Territory" or "input/output", so this code should technically be capitalized `cargoIOJSON` (and `getCreatesIOLatest`, etc)

Its probably cleaner in this context to drop the `Io` as you suggest elsewhere than try and apply this style.

Reply:
Good point, it is now renamed to just `Crates`.

Reply:
Thanks! I see you renamed `cratesPkgManager` below, but it looks like you may have missed a commit for the others instances of `cratesIo`.

Reply:
Ah my bad, forgot to press save in vscode, should be all renamed now.